cirq-core/cirq/sim/density_matrix_simulator_test.py

```diff
@@ -66,8 +67,10 @@ def test_invalid_dtype():
         cirq.DensityMatrixSimulator(dtype=np.int32)
 
 
-@pytest.mark.parametrize('dtype', [np.complex64, np.complex128])
-def test_run_with_ignore_measurement_results(dtype):
+@pytest.mark.parametrize(
```

Comment:
[Optional] Setting `dtype` and `split` in two separate `pytest.mark.parametrize` lines will have the same effect - pytest tests the Cartesian product of parameters defined that way.